cmd/skywire-cli/commands/config/gen.go

```diff
@@ -221,6 +285,46 @@ var genConfigCmd = &cobra.Command{
 				}
 			}
 		}
+		skywire := os.Args[0]
+		match := strings.Contains("/tmp/", skywire)
+		if (!stdout) || (!match) {
```

Comment:
Could we exclude windows here? means `selecteOS != "windows"`? Because we need `.exe` at file name there, otherwise cannot run (specific for vpn-client that use there currently).

Reply:
I fixed the behavior for windows. Have not directly tested but implemented as follows:

```
//binaries have .exe extension on windows
var exe string
if skyenv.OS == "windows" {
exe = ".exe"
}
// Disable apps not found at bin_path with above exceptions for go run and stdout
if _, err := os.Stat(conf.Launcher.BinPath + "/" + "skychat" + exe); err != nil {
```

so the string variable exe will just be an empty string unless the OS is windows

Reply:
I think is sufficient and OK. I'll test it when switching to windows after my current Erson redis layout testing,